src/main/java/util/DBConnection.java

```diff
@@ -0,0 +1,29 @@
+package util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+
+public class DBConnection {
+
+     public static Connection createConnection() {
+            Connection conn = null;
+            String url = "jdbc:mysql://localhost:3306/userapp";
+            String UserName = "root";
+            String password = "123456";
```

Comment:
This is a bad practice. Please don't use your credentials in the Java files. A few days ago I shared an option to do so:
https://www.baeldung.com/java-properties
https://www.geeksforgeeks.org/reading-and-writing-properties-file-in-java/